CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+### 0.8.1 
```

Comment:
This should probably be `0.9.0` since we're introducing new features and making a breaking change.